addons/medical_gui/functions/fnc_displayPatientInformation.sqf

```diff
@@ -1,244 +1,65 @@
 #include "script_component.hpp"
 /*
- * Author: Glowbal
- * Displays the patient information for given unit.
+ * Author: Glowbal, mharis001
+ * Opens the patient information display for given target.
  *
  * Arguments:
- * 0: The Unit <OBJECT>
- * 1: Show <NUMBER> (default: 0)
- * 2: Selection <NUMBER> (default: 0)
+ * 0: Target <OBJECT>
+ * 1: Body part <NUMBER>
  *
  * Return Value:
  * None
  *
  * Example:
- * [bob, true, 2] call ACE_medical_fnc_displayPatientInformation
+ * [_target, 0] call ace_medical_gui_fnc_displayPatientInformation
  *
  * Public: No
  */
-#define MAX_DISTANCE 10
 
-params ["_target", ["_show", 0], ["_selectionN", 0]];
+#define MAX_DISTANCE 4
```

Comment:
I experimented with distance a bit for ACRE2 external radios. It always gave me trouble around these values since it allowed for 0 movement. If  you tested and think it is fine,  ignore this comment.

Reply:
Do you mean that the distance is not enough? This controls how far the target has to move away for the patient info display to be hidden. 4 meters seems okay, haven't tested it fully though and can increase if needed.

Reply:
It may be. We will just keep it in mind